### PR TITLE
I-D text changes, CDDL, and example use construction described by Daniel Fett

### PR DESCRIPTION
**Pull request by rohanmahy**
Created a CDDL and example (in EDN) of the construction described by Daniel Fett, and made the draft consistent with that construction.

the KBT payload contains the a claim (called here issuer_sd_cwt) which is an embedded bstr of the entire Issuer SD-CWT. The unprotected header of the issued-cwt has zero or more disclosures.

I also moved sd_alg into the issuer SD-CWT protected header.

This construction dramatically simplified the CDDL schema. The example is a bit harder to read, merely because the issuer SD-CWT is more deeply nested. Overall this construction is simpler and requires fewer new claims.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: sd-cwt-example.cbor-diag [hunk 1]
```diff
@@ -1,76 +1,71 @@
...
-  } >>,
-  / signature / h'3337af2e66959614'
-])
\ No newline at end of file
+        / issuer_sd_cwt / 12 : << 18([
+          / protected / << {
+            / alg /    1  : -35, / ES384 /
+            / typ /    16 : "application/sd+cwt",
+            / kid /    4  : 'https://issuer.example/cwt-key3'
+            / sd_alg / 17 : -16  / SHA256 /
```
Comment (OR13) [inline sd-cwt-example.cbor-diag:16]: move back to payload

**Review by OR13 — CHANGES_REQUESTED** [review]
Need to move sd_alg back to payload, then we should merge this

**Review by mprorock — APPROVED** [review]

**Comment by mprorock** [comment]
Approved pending move of sd_alg

File: sd-cwt-example.cbor-diag (hunk 1 above)
Comment (rohanmahy) [reply sd-cwt-example.cbor-diag]: Fixed

**Comment by rohanmahy** [comment]
Moved sd_alg back to payload as requested.

**Pull request merged**